src/System.Data.SqlClient/tests/project.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+    "dependencies": {
+        "Microsoft.CSharp": "4.0.1-rc3-23722",
+        "System.Console": "4.0.0-rc3-23722",
```

Comment:
Removed. 
System.Console is no longer needed as I removed all console print outs from the test per our conversation yesterday.